metrics/prometheus.go

```diff
@@ -186,6 +186,38 @@ func NewPrometheusCollector(i infoProvider, f ContainerLabelsFunc, includedMetri
 					}
 					return values
 				},
+			}, {
+				name:        "container_cpu_user_seconds",
+				help:        "Per core time spent in user mode",
+				valueType:   prometheus.CounterValue,
+				extraLabels: []string{"cpu"},
```

Comment:
we should probably add this as a dimension to existing metrics, rather than create an entirely new one.